NEWS.rst

```diff
@@ -7,6 +7,24 @@
 
 .. towncrier release notes start
 
+20.1.1 (2020-05-19)
+===================
+
+Deprecations and Removals
+-------------------------
+
+- Drop parallelization from ``pip list --outdated``. (`#8167 <https://github.com/pypa/pip/issues/8167>`_)
+
+Bug Fixes
+---------
+
+- Revert building of local directories in place, restoring the pre-20.1
+  behaviour of copying to a temporary directory. (`#7555 <https://github.com/pypa/pip/issues/7555>`_)
```

Comment:
Do we want to advertise this revert as a bug fix?

Reply:
I too this that this is better suited for `Deprecations and Removals` section.

Reply:
Agreed! :)